README.md

```diff
@@ -703,10 +703,28 @@ Default: `true`
 
 #### omitSkippedTests ####
 
-Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCuucumberStyleTestReports is not set to `true`
+Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCucumberStyleTestReports is not set to `true`
 
 Default: `true`
 
+#### pylintDisabledChecker ####
+
+Controls the disabled checkers for Pylint. 
+
```

Comment:
S (Substantive): what do these letters mean and why don't we make them more user friendly?

Reply:
Perhaps b/c of these options - but why not just have this be `lintEnabled=true/false`?

Reply:
The pylintDisabledChecker (now referenced as lintDisabledChecker) is used to configure the checkers to disable on pylint. Likewise, the pylineEnabledChcker (now referenced as lintEnabledChecker) is used to configure the checkers to enable on pylint (checkers referring to https://pylint.readthedocs.io/en/latest/user_guide/checkers/features.html). Hence, both configurations are not used for enabling/disabling the linting package, but to configure which checkers to enable/disable for linting. 

But to your first comment, yup, I can add a small description to the default values, as the characters are not obvious unless reader has used Pylint before.

Reply:
We probably do want an on/off control for linting in general

Reply:
Added the configurations: lintSource and lintTest, that controls linting for the source and test modules respectively. Thank you for the suggestion.